integration/setup/setup.go

```diff
@@ -257,23 +260,25 @@ func installHostPeerVPC(config Config) (string, error) {
 	return vpcPeerID, nil
 }
 
-func installResources(config Config, vpcPeerID string) error {
+func installResources(ctx context.Context, config Config, vpcPeerID string) error {
 	var err error
 
 	{
-		// TODO configure chart values like for the other operators below.
-		err = config.Host.InstallStableOperator("cert-operator", "certconfig", e2etemplates.CertOperatorChartValues)
+		certOperatorChartValues := os.ExpandEnv(e2etemplates.CertOperatorChartValues)
+		err = config.Release.InstallOperator(ctx, "cert-operator", release.NewStableVersion(), certOperatorChartValues, corev1alpha1.NewCertConfigCRD())
 		if err != nil {
 			return microerror.Mask(err)
 		}
-		err = config.Host.InstallStableOperator("node-operator", "drainerconfig", e2etemplates.NodeOperatorChartValues)
+
+		nodeOperatorChartValues := os.ExpandEnv(e2etemplates.NodeOperatorChartValues)
+		err = config.Release.InstallOperator(ctx, "node-operator", release.NewStableVersion(), nodeOperatorChartValues, corev1alpha1.NewNodeConfigCRD())
 		if err != nil {
 			return microerror.Mask(err)
 		}
```

Comment:
We still need to fix `ExpandEnv` here by moving by creating new chartvalues in e2etemplates.

Reply:
Fixed 🎉